### DISCUSSION TITLE
crop on TIFF giving multipage output

### DISCUSSION THREAD

**Discussion by jpnp** [ImageMagick/ImageMagick#5246] [Help] [answered]
I'm attempting to use `-crop` to extract a strip from the side of a TIFF image, but what I get is a multipage TIFF where the whole thing has been cut into strips and included in the output:

```
identify plain-test1.tif
plain-test1.tif TIFF 1122x780 1122x780+24+24 8-bit CMYK 47257B 0.020u 0:00.000

convert plain-test1.tif +repage -crop 128x780 -rotate 90 out.tif

identify out.tif 
out.tif[0] TIFF 780x128 780x128+0+0 8-bit CMYK 52469B 0.000u 0:00.010
out.tif[1] TIFF 780x128 780x128+0+128 8-bit CMYK 0.000u 0:00.010
out.tif[2] TIFF 780x128 780x128+0+256 8-bit CMYK 0.000u 0:00.010
out.tif[3] TIFF 780x128 780x128+0+384 8-bit CMYK 0.000u 0:00.010
out.tif[4] TIFF 780x128 780x128+0+512 8-bit CMYK 0.000u 0:00.000
out.tif[5] TIFF 780x128 780x128+0+640 8-bit CMYK 0.000u 0:00.000
out.tif[6] TIFF 780x128 780x128+0+768 8-bit CMYK 0.000u 0:00.000
out.tif[7] TIFF 780x128 780x128+0+896 8-bit CMYK 0.000u 0:00.000
out.tif[8] TIFF 780x98 780x98+0+1024 8-bit CMYK 0.000u 0:00.000
```
Do you know why this happens?

**Comment by fmw42** [comment, marked as answer]
You are using the tile cropping syntax, when you do not specify the +X+Y offsets in the crop. So you get strips offset by the width and height. 

If you want to remove strips on the sides, then use -shave or -chop or crop once on each side.

See https://imagemagick.org/Usage/crop/

**Reply by jpnp** [reply]
Aha, `-crop 128x780+0+0` does what I expected.  Thanks.
